sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java

```diff
@@ -505,6 +619,30 @@ public EditCaseSteps(
                   "placeDescription"));
         });
 
+    When(
+        "I am checking all Exposure data is saved and displayed in Cases",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(OPEN_SAVED_EXPOSURE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(OPEN_SAVED_EXPOSURE_BUTTON);
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(START_OF_EXPOSURE_INPUT);
+          String contactToCaseUIvalue =
+              (webDriverHelpers.getValueFromCombobox(CONTACT_TO_CASE_COMBOBOX)).toUpperCase();
+          String contactToCase =
+              apiState.getLastCreatedPerson().getFirstName().toUpperCase()
+                  + " "
+                  + apiState.getLastCreatedPerson().getLastName().toUpperCase()
+                  + " "
+                  + "("
+                  + dataOperations.getPartialUuidFromAssociatedLink(
+                      apiState.getCreatedContact().getUuid().toUpperCase())
+                  + ")";
+
+          softly.assertEquals(contactToCase, contactToCaseUIvalue, "The value is different");
```

Comment:
please change the assert message to reflect the problem in case of failure, the message is to ambiguous